src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -186,14 +186,20 @@ Supported in Sentry Android Gradle Plugin version `3.0.0` and above.
 
 </Note>
 
-The [Sentry Android Gradle Plugin](/platforms/android/proguard/#gradle-configuration) does the tracing auto instrumentation via bytecode manipulation for `SQLite` and `Room` libraries.
+The [Sentry Android Gradle Plugin](/platforms/android/proguard/#gradle-configuration) does the tracing auto instrumentation via bytecode manipulation for `androidx.sqlite` and `androidx.room` libraries.
 
 The Plugin injects a code snippet that starts a span out of the active span bound to the scope for each `CRUD` operation. The SDK sets the span `operation` to `db` and `description` to the SQL Query if available.
 
 The span finishes once the operation has been executed. The span `status` is set to `SpanStatus.OK` if successful or `SpanStatus.INTERNAL_ERROR` if there was any error.
 
 When the operation throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and SentryEvent will be linked when viewing it on the **Issue Details** page in sentry.io.
 
+<Note>
+ 
+At the moment, only a standard `androidx.room` usage is supported, i.e. SQL queries for any other `SupportSQLiteOpenHelper.Factory` than [androidx.sqlite](https://github.com/androidx/androidx/tree/androidx-main/sqlite) will not be reported. However, it's worth reporting an [issue](https://github.com/getsentry/sentry-android-gradle-plugin/issues), as we could consider adding a new integration.
```

Comment:
It would be nice to use the active voice to state the subject clearly. When using the passive voice, you can leave out the subject, and you don't have to be specific. I think it is essential to be clear when writing docs.

Reply:
I am not sure what exactly you are referring to?

Reply:
To @philipphofmann's first point, I've changed the first sentence to active voice. So I'm stating that "we (Sentry)" only support standard usage, rather than it not being clear who is or isn't supporting standard usage. I don't know if this is technically accurate or not but just demonstrating how that might look. Having said that, sometimes there are good use cases for passive voice. I'm not sure if the "who" in the second sentence is the SDK or something else, so I don't know if it makes sense to change that sentence or not. 

```suggestion
At the moment, we only support standard `androidx.room` usage. For example, SQL queries for any `SupportSQLiteOpenHelper.Factory` other than [androidx.sqlite](https://github.com/androidx/androidx/tree/androidx-main/sqlite) will not be reported. However, it's worth reporting an [issue](https://github.com/getsentry/sentry-android-gradle-plugin/issues), as we might consider adding a new integration.
```

Reply:
Aha, that's what was meant, gotcha. Thanks for the suggestion 👍

Reply:
yeah in the 2nd sentence it's the SDK "SDK will not report SQL queries for any `blah` other than ....`